composer.json

```diff
@@ -43,7 +47,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.4.x-dev"
+            "dev-master": "2.0.x-dev"
```

Comment:
we can always create a branch from the latest 1.4 tag if there is
something we need to fix urgently.